### PR TITLE
publish page changed event to the window

### PR DESCRIPTION
**Pull request by dutyuzh**
publish page changed event to the window

### CODE REVIEW

File: src/infinite-scroll.js [hunk 1]
```diff
@@ -222,6 +222,10 @@ export class InfiniteScroll {
     }
 
     _checkIfPageChanged() {
+
+        // Create the event
+        const pageChangedEvent = new Event('pageChangedEvent');
```
Comment (itmayziii) [inline src/infinite-scroll.js]: I do not think this is supported in IE?

**Review by itmayziii — CHANGES_REQUESTED** [review]
I do not believe this is supported in IE.

File: src/infinite-scroll.js (hunk 1 above)
Comment (itmayziii) [reply src/infinite-scroll.js]: I would suggest either using "The old fashioned way" https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Creating_and_triggering_events

Comment (dutyuzh) [reply src/infinite-scroll.js]: IE 11 + ... are we supporting older versions?

**Review by itmayziii — APPROVED** [review]

**Pull request merged**